eventstore/store_billable_events_test.go

```diff
@@ -1137,4 +1167,502 @@ var _ = Describe("GetBillableEvents", func() {
 			},
 		}))
 	})
+
+	/*-----------------------------------------------------------------------------------*
+	       00:00       01:00                                                             .
+	         |           |                                                               .
+	 .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .
+	 .   .   [====db1====]   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .
+	 .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .
+	       start       stop                                                  .
+	 .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .
+	<=======================================PLAN1 (cf_id)===============================>.
+	 .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .   .
+	-------------------------------------------------------------------------------------*/
+	It("Should include BillableEvent with event.plan_guid == pricing_plan.plan_guid", func() {
```

Comment:
This isn't helpful text. It should describe behaviour, not implementation.

Reply:
I will think something better.  Anyway, the test is quite low level